### DISCUSSION TITLE
2022/05/19/table-of-contents-with-sanity-portable-text/

### DISCUSSION THREAD

**Discussion by giscus** [KittyGiraudel/site#160] [Announcements]
# 2022/05/19/table-of-contents-with-sanity-portable-text/

I’m Kitty Giraudel, a non-binary trans frontend developer based in Berlin, focused on accessibility and inclusivity.

https://kittygiraudel.com/2022/05/19/table-of-contents-with-sanity-portable-text/

**Comment by alex-suprun-git** [comment]
Thanks for sharing! Very glad to find your tutorial
